benchmarks/helics/PholdFederate.hpp

```diff
@@ -26,21 +26,29 @@ class PholdFederate
 
     int index_ = 0;
     int maxIndex_ = 0;
-    int initEvCount_ = 16; // starting number of events
+    unsigned int initEvCount_ = 16; // starting number of events
     double localProbability_ = .1; // probability of local events
     double randTimeMean_ = deltaTime * 2; // mean for the exponential distribution used when picking event times
 
-    // classes related to the exponential distribution random number generator
-    std::random_device rd;
+    // classes related to the exponential and uniform distribution random number generator
+    bool generateRandomSeed = true; 
+    unsigned int seed = 0;
```

Comment:
Generally a seed of 0 is not the best idea.  My guess is it is fine, but some other numbers might be a better choice here.  pretty sure <random> handles it properly, but would prefer some defined (non-zero number) as the default.

Reply:
I found some suggestions that not having the majority of the bits be 0 in the seed might make the generated numbers "better". The alternate choice, 0x600d5eed had quite a few more 0 bits.